### PR TITLE
(BOLT-992) update error message for winrm timeout

### PR DESCRIPTION
**Pull request by adreyer**

### CODE REVIEW

**Comment by nicklewis** [comment]
rubocop is displeased

File: lib/bolt/transport/winrm/connection.rb [hunk 1]
```diff
@@ -58,7 +58,8 @@ def connect
           # If we're using the default port with SSL, a timeout probably means the
           # host doesn't support SSL.
           if target.options['ssl'] && @port == HTTPS_PORT
-            the_problem = "\nUse --no-ssl if this host isn't configured to use SSL for WinRM"
+            the_problem = "Verify that required WinRM ports are open, " \
```
Comment (nicklewis) [inline lib/bolt/transport/winrm/connection.rb]: Looks like we need the `\n` here.

```
Started on wwgmxp5nablefwt.delivery.puppetlabs.net...
Failed on wwgmxp5nablefwt.delivery.puppetlabs.net:
  Timeout after 10 seconds connecting to https://wwgmxp5nablefwt.delivery.puppetlabs.net:5986/wsmanVerify that required WinRM ports are open, or use --no-ssl if this host isn't configured to use SSL for WinRM.
Failed on 1 node: winrm://Administrator:Qu@lity!@wwgmxp5nablefwt.delivery.puppetlabs.net
Ran on 1 node in 10.17 seconds
```

**Comment by nicklewis** [comment]
I used `winrm delete winrm/config/Listener?Address=*+Transport=HTTPS` to disable WinRM SSL to test this.

**Comment by donoghuc** [comment]
I used `PS C:\cygwin64\home\Administrator> New-NetFirewallRule -DisplayName 'Windows Remote Management (HTTPS-In)' -Direction Inbound -Protocol TCP -LocalPort 5986 -Action Block | Format-List` 
```
cas@cas-ThinkPad-T460p:~/working_dir/bolt$ bolt command run whoami -n override-winrm
Started on ah0ggldnzsaahhd.delivery.puppetlabs.net...
Failed on ah0ggldnzsaahhd.delivery.puppetlabs.net:
  Timeout after 10 seconds connecting to https://ah0ggldnzsaahhd.delivery.puppetlabs.net:5986/wsmanVerify that required WinRM ports are open, or use --no-ssl if this host isn't configured to use SSL for WinRM.
Failed on 1 node: ah0ggldnzsaahhd.delivery.puppetlabs.net
Ran on 1 node in 10.12 seconds
```
Duplicating effort.

**Pull request merged**